README.md

```diff
@@ -8,11 +8,11 @@ By default, it takes `TEMPLATE.md` and outputs `README.md`.
 
 ## Inputs
 
-| name         | type   | default         | description |
-| ------------ | ------ | --------------- | ----------- |
-| token        | string |                 | GitHub personal access token used to fetch data. Pass a secret by for instance using `${{ secrets.README_TEMPLATE_TOKEN }}`. [Go here](https://github.com/settings/tokens/new?scopes=read:user) to generate one with the `read:user` scope
-| template     | string | `"TEMPLATE.md"` | Template file path
-| readme       | string | `"README.md"`   | Output file path
+| name         | required | type   | default         | description |
```

Comment:
I think it is important to mention that these params are mandatory. 

An alternative to adding a column called `required` could be to just write a sentence saying that they are required. I am opened to suggestions.

Reply:
Sure, that works